csireverseproxy/pkg/config/config.go

```diff
@@ -76,6 +78,8 @@ type StorageArray struct {
 	PrimaryEndpoint        url.URL
 	SecondaryEndpoint      url.URL
 	ProxyCredentialSecrets map[string]ProxyCredentialSecret
+	Labels                 map[string]string
+	Parameters             map[string]string
```

Comment:
We probably need to add constants for the current set of parameters, as shown in the sample secret.yaml. What are other drivers doing? Do they use a dictionary or have a new struct with defined fields?

Reply:
ok will do in the next PR. thanks for the comment.